package-lock.json

```diff
@@ -2512,11 +2512,6 @@
         "regenerator-runtime": "^0.11.0"
       },
       "dependencies": {
-        "core-js": {
```

Comment:
Were these removed intentionally? Or npm did that?

Reply:
That was npm, not an intentional change by me